website/auth.py

```diff
@@ -113,6 +113,8 @@ def login():
         if not check_password(body['password'], user['password']):
             return 'invalid username/password', 403
 
+        session['user'] = user
```

Comment:
This should become a slice of the data.

I would prefer to automatically invalidate `is_teacher` (based on time).

Reply:
The following slice of the data is placed in the current session
* `username`
* `email`
* `is_teacher`

Reply:
I have implemented a time-to-live for the current user